src/code/z_sram_NES.c

```diff
@@ -99,7 +99,7 @@ u16 sPersistentCycleWeekEventRegs[ARRAY_COUNT(gSaveContext.save.saveInfo.weekEve
     /* 22 */ PERSISTENT_WEEKEVENTREG(WEEKEVENTREG_22_02) |
         PERSISTENT_WEEKEVENTREG(WEEKEVENTREG_RECEIVED_HONEY_AND_DARLING_HEART_PIECE),
     /* 23 */ PERSISTENT_WEEKEVENTREG(WEEKEVENTREG_OBTAINED_GREAT_SPIN_ATTACK) |
-        PERSISTENT_WEEKEVENTREG(WEEKEVENTREG_RECEIVED_BEAVER_BROS_BOTTLE),
+        PERSISTENT_WEEKEVENTREG(WEEKEVENTREG_RECEIVED_BEAVER_RACE_BOTTLE),
```

Comment:
It wasn't touched by this PR, but I wonder if `WEEKEVENTREG_OBTAINED_GREAT_SPIN_ATTACK` could be `WEEKEVENTREG_RECEIVED_GREAT_SPIN_ATTACK` for consistency?

Reply:
Good catch, I missed that one. Let me change it.